tests/integration/timeout_test.py

```diff
@@ -55,5 +55,5 @@ def test_lot_of_vlans_with_bridges(eth1_up):
             ]
         )
     checkpoint = libnmstate.apply({Interface.KEY: interfaces}, commit=False)
-    libnmstate.rollback(checkpoint)
+    libnmstate.rollback(checkpoint=checkpoint)
```

Comment:
Good, please keep this as its own commit but reorder it to be the first commit. Also follow the commit template from the repo.